README.rst

```diff
@@ -29,38 +48,25 @@ Installation
         ftw.linkchecker
 
 
-One needs to add a config file (e.g. linkchecker_config.json) holding:
+Configuration
+=============
+
+A config file is required (there is an example in ``config.example``).
+In the config file following things can be configured:
 
 - portal path (unique identifier of the platform)
 - emails of the platforms administrator (the ones who gets the report)
-- base URI (domain where the platform is configured)
+- base URI (domain where the platform is configured - it will be used to prepend in the report)
 - timeout in seconds (how long the script waits for each external link before
   continuing if the page does not respond).
-- upload_location can be left empty. It is the path to a file listing
-  block where the report will be uploaded.
-
-::
-
-    {
-      "/portal/path-one": {
-        "email": ["first_site_admin@example.com", "first_site_keeper@example.com"],
-        "base_uri": "http://example1.ch",
-        "timeout_config": "1",
-        "upload_location": "/content_page/my_file_listing_block"
-      },
-      "/portal/path-two": {
-        "email": ["second_site_admin@example.com"],
-        "base_uri": "http://example2.ch",
-        "timeout_config": "1"
-      }
-    }
-
+- upload_location can be left empty. It is the path to a ``ftw.simplelayout`` file listing
+  block where the report will additionally be uploaded.
```

Comment:
Why do we need ftw.simplelayout in order to upload?  It should be uploadable anywhere.... :-)

Reply:
I create issues regarding this so we can go on with this PR.